cmake/SearchForStuff.cmake

```diff
@@ -111,3 +111,10 @@ else()
   set(IGN_MATH_VER ${ignition-math6_VERSION_MAJOR})
   message(STATUS "Looking for ignition-math${IGN_MATH_VER}-config.cmake - found")
 endif()
+
+########################################
+# Find ignition cmake2
+# Only for using the testing macros, not really
+# being use to configure the whole project
+find_package(ignition-cmake2 2.3 REQUIRED)
```

Comment:
does this need to be called before finding `ignition-math6`? it's a dependency of `ignition-math6`, so I'm not sure if an old version would be improperly cached